### ISSUE TITLE
Pyyaml 5.4 caused building wheels error - Linux

### ISSUE THREAD

**Issue by M-Atkins** [akai-katto/dandere2x#297]
Running `pip3 install -r requirements.txt` caused "Failed building wheel" error.

Installing the latest version of pyyaml with `pip install pyyaml` fixed this.

Also changing:
```
#include "Block.h"
```
To:
```
#include "Block.h"
#include <cstdint>
```

In `dandere2x_cpp/plugins/block_plugins/Block.cpp` was necessary to get installation working on my system (Arch Linux)

**Comment by KRDucky** [comment]
yeah, there are CPP memory leaks etc in the files. Had to fix them to get even the cpp part to compile. 
Ive linked to the fixes. 
[DiamondSearch CPP File](https://github.com/KRDucky/fixes/blob/main/DiamondSearch.cpp)
[DiamondSearch H file](https://github.com/KRDucky/fixes/blob/main/DiamondSearch.h)
